modules/cnf-running-the-performance-creator-profile-hosted.adoc

```diff
@@ -0,0 +1,136 @@
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                     CLUSTER       DESIRED NODES   CURRENT NODES   AUTOSCALING   AUTOREPAIR   VERSION   UPDATINGVERSION   UPDATINGCONFIG   MESSAGE
+democluster-us-east-1a   democluster   1               1               False         False        4.17.0    False             False            
+----
+
+. Use Podman to authenticate to `registry.redhat.io` by running the following command:
++
+[source,terminal]
+----
+$ podman login registry.redhat.io
+----
++
+[source,bash]
+----
+Username: <user_name>
+Password: <password>
+----
+
+. Create a performance profile by running the following command. The example uses sample PPC arguments and values:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run --entrypoint performance-profile-creator \
+    -v /path/to/must-gather:/must-gather:z \
+    registry.redhat.io/openshift4/ose-cluster-node-tuning-rhel9-operator:v{product-version} \
+    --must-gather-dir-path /must-gather \
+    --reserved-cpu-count=1 \
+    --rt-kernel=true \
+    --split-reserved-cpus-across-numa=false \
+    --topology-manager-policy=single-numa-node \
+    --node-pool-name=democluster-us-east-1a \
+    --power-consumption-mode=ultra-low-latency \
+    --offlined-cpu-count=1 > my-hosted-cp-performance-profile.yaml
+----
++
+* `-v <path_to_must_gather>` specifies the path to either of the following components:
+** The directory containing the `must-gather` data.
+** The directory containing the `must-gather` decompressed .tar file.
+* `--reserved-cpu-count=1` specifies one reserved CPU.
+* `--rt-kernel=true` enables the real-time kernel.
+* `--split-reserved-cpus-across-numa=false` disables reserved CPUs splitting across NUMA nodes.
+* `--topology-manager-policy=single-numa-node` specifies the NUMA topology policy. This must be set to 1 if installing the installing the NUMA Resources Operator. 
+* `--power-consumption-mode=ultra-low-latency` specifies minimal latency at the cost of increased power consumption.
+* `--offlined-cpu-count=1` specifies one offlined CPU.
++
+.Example output
+[source,terminal]
+----
+
+----
+// Can't the MCP name be whatever the user wants, regardless of SNO vs multi-mode?
+
+. Review the created YAML file by running the following command:
++
+[source,terminal]
+----
+$ cat my-hosted-cp-performance-profile
+----
+.Example output
++
+[source,yaml]
+----
+---
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  cpu:
+    isolated: 1,3-11,13,15-23
+    reserved: 0,2,12,14
+  nodeSelector:
+    node-role.kubernetes.io/worker: ''
+  numa:
+    topologyPolicy: single-numa-node
+  realTimeKernel:
+    enabled: false
+  workloadHints:
+    highPowerConsumption: true
+    perPodPowerManagement: false
+    realTime: true
+----
++
+You need to embed this profile in a `ConfigMap` and reference it in your `NodePools` to apply it to the hosted cluster.
```

Comment:
Nope. if PPC detects the MG is from HyperShift it will embed the PerformanceProfile in a ConfigMap already